forms-html/sara_abdulbasit/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Form</title>
+</head>
+
+<body>
+  <li>make the email input required</li>
+  <br>
+  <br>
+  <form>
+    <label for="email">ENTER YOUR EMAIL</label> <br>
+    <input type="email" id="email" name="email" value="osama@re-coded.com"><br>
```

Comment:
the email should be required, when the user clicks the submit button it should ask him to fill it if it's empty